tests/foreman/api/test_operatingsystem.py

```diff
@@ -66,14 +155,32 @@ def test_point_to_arch(self):
         self.assertEqual(operating_sys.architecture[0].id, arch.id)
 
     @run_only_on('sat')
-    def test_point_to_ptable(self):
+    def test_positive_create_with_multiple_archs(self):
+        """@Test: Create an operating system that points at multiple different
+        architectures.
+
+        @Feature: Operating System
+
+        @Assert: The operating system is created and points at the expected
+        architectures.
+        """
+        amount = range(random.randint(3, 5))
```

Comment:
I'm curios why are we using both `random.randint` and `fauxfactory.gen_integer` in the same module. Mind dropping random dependency and update calls with `gen_integer()`? Not a blocker